packages/core/src/Stores/ui-store.js

```diff
@@ -785,8 +784,8 @@ export default class UIStore extends BaseStore {
         this.is_update_email_modal_visible = state_change;
     }
 
-    setResetTradingPasswordModalOpen(is_reset_trading_password_modal_visible) {
-        this.is_reset_trading_password_modal_visible = is_reset_trading_password_modal_visible;
+    setResetTradingPasswordModalOpen(val) {
+        this.is_reset_trading_password_modal_visible = val;
     }
```

Comment:
why do we need this change

Reply:
it's just making the name shorter and not confusing an external param with a variable in the store

Reply:
Shorter value which is not descriptive is not preferable. The previous value was better.